robots.yaml

```diff
@@ -0,0 +1,23 @@
+bender:
```

Comment:
Hodil by se komentar-popis, k cemu ten soubor je. Napr.
_Change displayed robot's name for the one you like. Place the name of your choice instead of listed **displayed_name**, make sure to write it between the quotation marks. Robots appear on the board in the same order as they are listed in the file. Recommended name length is max. 8 characters, a longer one won't look well on the interface screen._

Reply:
Děkuju